tests/tests-core/src/main/java/com/gentics/mesh/core/field/s3binary/S3BinaryFieldEndpointTest.java

```diff
@@ -259,19 +335,71 @@ public void testUploadWorking() {
             call(() -> client().updateNodeS3BinaryField(
                     PROJECT_NAME,
                     s3binaryNode.getUuid(),
-                    "s3",
+                    FIELD_NAME,
                     UPLOAD_REQUEST
             ));
             //uploading
             File tempFile = createTempFile();
             s3BinaryStorage().createBucket("test-bucket").blockingGet();
-            s3RestResponse1 = s3BinaryStorage().uploadFile("test-bucket", s3binaryNode.getUuid() + "/s3", tempFile, false).blockingGet();
+            s3RestResponse1 = s3BinaryStorage().uploadFile("test-bucket", s3binaryNode.getUuid() + "/s3/en", tempFile, false).blockingGet();
         }
-        Boolean doesObjectExists = s3BinaryStorage().exists("test-bucket", s3binaryNode.getUuid() + "/s3").blockingGet();
+        Boolean doesObjectExists = s3BinaryStorage().exists("test-bucket", s3binaryNode.getUuid() + "/s3/en").blockingGet();
         assertTrue(doesObjectExists);
         assertNotNull(s3RestResponse1);
+        NodeResponse s3binaryNode1 = call(() -> client().findNodeByUuid(PROJECT_NAME, s3binaryNode.getUuid()));
+        assertNotNull(s3binaryNode1);
+        assertNotNull(s3binaryNode1.getFields().getS3BinaryField(FIELD_NAME));
     }
 
+    @Test
+    public void testUploadOldFormatWorking() {
+        S3RestResponse s3RestResponse = new S3RestResponse();
+        s3RestResponse.setVersion("1");
+        NodeResponse s3binaryNode = createNodeWithField();
+        S3RestResponse s3RestResponse1 = null;
+        //creating
+        call(() -> client().updateNodeS3BinaryField(
+                PROJECT_NAME,
+                s3binaryNode.getUuid(),
+                FIELD_NAME,
+                UPLOAD_REQUEST
+        ));
+        // some very illegal stuff
+        try (Tx tx = tx()) {
```

Comment:
We have explicitly disallowed creation S3 binaries the old way, but somehow this has to be tested.